dashboard/test/factories/reference_guides.rb

```diff
@@ -0,0 +1,22 @@
+# == Schema Information
+#
+# Table name: reference_guides
+#
+#  id                :bigint           not null, primary key
+#  name              :string(255)
+#  slug              :string(255)
+#  course_version_id :integer
+#  content           :text(65535)
+#  order             :integer
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#
+FactoryGirl.define do
```

Comment:
I'm not sure why we have this practice but in general we have been keeping all new factories in factories.rb instead of making their own file. @davidsbailey thoughts?

Reply:
sorry, i missed in the migration PR that this topic hadn't quite been settled. it is a good point that the factories.rb file is getting huge, and there isn't any work planned to split it up. I did notice there is a pd_factories.rb file, which could be an interesting example to follow. what do folks think, should there be a curriculum_factories.rb, for everything from course offering down to script_level?

Reply:
That sounds good to me